packages/rocketchat-setup-wizard/client/setupWizard.js

```diff
@@ -0,0 +1,268 @@
+import s from 'underscore.string';
+
+const setSettingsAndGo = (settings, registerServer = true) => {
+	const settingsFilter = Object.entries(settings)
+		.filter(key => !/registration-|registerServer|currentStep/.test(key))
+		.map(setting => {
+			return {
+				_id: setting[0],
+				value: setting[1]
+			};
+		});
```

Comment:
Why not?
```
.map(([_id, value]) => return {_id, value})
```

Reply:
We can't do that because we don't have an object with `_id` and `value`... we have an array with two strings, the first one is the `_id` and the second is the `value`.

Reply:
@karlprieb please reevaluate my suggestion